test/DBSeed/Departments.sql

```diff
@@ -1,71 +1,70 @@
-INSERT INTO Departments (DepartmentID, Name, ParentDepartmentID) VALUES (1, 'Activities', 1);
```

Comment:
Since we're already in here updating this file anyway, could we update the names of a few of these departments? For example, I think "Smoker's Paradise" was renamed recently, wasn't it?

If we want to not make those updates in this PR, but instead in another, that would be fine too.